File: DummyData/gen-seq-data.pl
#!/usr/bin/env perl

use strict;
use warnings;

if (@ARGV != 2) {
  printf("Usage: %s FILENAME SIZE\n", $0);
  exit(1)
}

my $fname = $ARGV[0];
my $end = ($ARGV[1] / 4) - 1;

open(my $fh, ">$fname") || die "Failed to open: $fname: $!";
for my $i (0..$end) {
  $fh->print(pack("I", $i));
}

File: DummyData/gen-seq-data.py
#!/usr/bin/env python3
## -*- coding: utf-8 -*- vim:shiftwidth=4:expandtab:

import sys

if len(sys.argv) != 3:
    print(f'Usage: {sys.argv[0]} FILENAME SIZE')
    exit(1)

fname = sys.argv[1]
stop = int(sys.argv[2]) // 4

with open(fname, 'wb') as f:
    for b in range(stop):
        f.write(b.to_bytes(4))
